src/core/annotation.js

```diff
@@ -1427,7 +1435,28 @@ class WidgetAnnotation extends Annotation {
 
     data.readOnly = this.hasFieldFlag(AnnotationFieldFlag.READONLY);
     data.required = this.hasFieldFlag(AnnotationFieldFlag.REQUIRED);
-    data.hidden = this._hasFlag(data.annotationFlags, AnnotationFlag.HIDDEN);
+
+    const print = this._hasFlag(data.annotationFlags, AnnotationFlag.PRINT);
+    const hidden = this._hasFlag(data.annotationFlags, AnnotationFlag.HIDDEN);
+    const noView = this._hasFlag(data.annotationFlags, AnnotationFlag.NOVIEW);
```

Comment:
Can we not use the shorter `this.hasFlag(AnnotationFlag.WHATEVER)` format instead here?